buildenv

```diff
@@ -118,7 +125,40 @@ EOF
 zopen_post_install()
 {
   # Install a cacert.pem to be used (optionally) by the customer
-  zopen update-cacert -f $1
+  if ! zopen update-cacert -f $1 ; then
+    return 4
```

Comment:
I don't think we check the return code for post-install - https://github.com/ZOSOpenTools/meta/blob/main/bin/lib/zopen-build#L1339. Maybe we should? You could also call the `printError` function

Reply:
Will do. Yeah - we should check the return code.

Reply:
added printSoftError so that code continues and the caller can 'fail out' if necessary

Reply:
added check to post_install in the manpagetweak PR: https://github.com/ZOSOpenTools/meta/pull/239